src/itemset.js

```diff
@@ -115,11 +115,13 @@ define([
                     var possibleSrcs = _.map(datasources.getPossibleFixtures(), 
                                              function(val) { return val.src; }),
                         notCustom = _.contains(possibleSrcs, itemsetData.instance.src),
-                        choices = datasources.autocompleteChoices(itemsetData.instance.src);
+                        choices = datasources.autocompleteChoices(itemsetData.instance.src),
+                        strippedValue = itemsetData.valueRef.replace(/(\[.+])+/, ""),
+                        strippedLabel = itemsetData.labelRef.replace(/(\[.+])+/, "");
```

Comment:
I changed the regex to `/\[[^\[]+]/g`

Using your examples:
* `bar[a=1][b=2]` becomes `bar`
* `bar[a=1]/foo[b=2]` becomes `bar/foo`
* `bar[a=1]/foo` becomes `bar/foo`

I think this is desired, because we will want to make sure that bar/foo is an valid attribute in the structure.